### PR TITLE
clear up dockerfile

### PR DESCRIPTION
**Pull request by jiazhenjiang**
clear up dockerfile

### CODE REVIEW

File: ci/images/README.md [hunk 1]
```diff
@@ -45,21 +45,21 @@ tugraph/tugraph-db-[os name & version]:[tugraph-db version]
 
 ### 创建"TuGraph Compile Image"
 
-不同操作系统版本的Dockerfile在 `ci/images/`目录下，目前支持 `ubuntu16.04`、`ubuntu18.04`、`centos7.3`。
+不同操作系统版本的Dockerfile在 `ci/images/`目录下，根据需要选择compile或runtime版本。
```
Comment (knightast) [inline ci/images/README.md:48]: 目前提供centos7版本

File: ci/images/README.md [hunk 2]
```diff
@@ -69,14 +69,14 @@ docker tugraph/tugraph-env-centos7:1.1.0 .
 - DataX_Path: DataX路径。
```
Comment (knightast) [inline ci/images/README.md:69]: TuGraphEnvDockerImage -> CompileDockerImage
TuGraphDockerImage -> RuntimeImage

File: ci/images/compile/centos-7-Dockerfile [hunk 3]
```diff
@@ -38,9 +37,9 @@ RUN wget http://ftp.gnu.org/gnu/gcc/gcc-5.4.0/gcc-5.4.0.tar.gz \
 RUN wget https://www.python.org/ftp/python/3.6.9/Python-3.6.9.tgz \
     && tar zxf Python-3.6.9.tgz && cd Python-3.6.9 && ./configure --prefix=/usr/local \
     && make -j4 && make install && python3 -m pip install pexpect requests && rm -rf /Python*
-# install cmake-3.13.4
-RUN wget https://cmake.org/files/v3.13/cmake-3.13.4.tar.gz \
-    && tar xzf cmake-3.13.4.tar.gz && cd cmake-3.13.4 \
+# install cmake
+RUN wget https://cmake.org/files/v3.14/cmake-3.14.1.tar.gz \
+    && tar -zxvf cmake-3.14.1.tar.gz && cd cmake-3.14.1 \
```
Comment (knightast) [inline ci/images/compile/centos-7-Dockerfile:42]: tar xvf

File: ci/images/README.md (hunk 1 above)
Comment (jiazhenjiang) [reply ci/images/README.md]: already done

File: ci/images/README.md (hunk 2 above)
Comment (jiazhenjiang) [reply ci/images/README.md:69]: already done

File: ci/images/compile/centos-7-Dockerfile (hunk 3 above)
Comment (jiazhenjiang) [reply ci/images/compile/centos-7-Dockerfile]: already done

File: ci/images/compile/centos-7-Dockerfile [hunk 4]
```diff
@@ -38,9 +37,9 @@ RUN wget http://ftp.gnu.org/gnu/gcc/gcc-5.4.0/gcc-5.4.0.tar.gz \
 RUN wget https://www.python.org/ftp/python/3.6.9/Python-3.6.9.tgz \
     && tar zxf Python-3.6.9.tgz && cd Python-3.6.9 && ./configure --prefix=/usr/local \
     && make -j4 && make install && python3 -m pip install pexpect requests && rm -rf /Python*
-# install cmake-3.13.4
-RUN wget https://cmake.org/files/v3.13/cmake-3.13.4.tar.gz \
-    && tar xzf cmake-3.13.4.tar.gz && cd cmake-3.13.4 \
+# install cmake
+RUN wget https://cmake.org/files/v3.14/cmake-3.14.1.tar.gz \
+    && tar xvf cmake-3.14.1.tar.gz && cd cmake-3.14.1 \
```
Comment (knightast) [inline ci/images/compile/centos-7-Dockerfile:42]: xf

File: ci/images/runtime/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,23 @@
+ARG TuGraphEnvDockerImage
```
Comment (knightast) [inline ci/images/runtime/Dockerfile:1]: TuGraphEnvDockerImage  -> CompileDockerImage

**Comment by knightast** [comment]
Have you push the new docker to dockerhub? If so, update the dockerhub url in docs.

**Review by knightast — APPROVED** [review]

**Pull request merged**